src/gui/MainWindow.cpp

```diff
@@ -194,9 +195,13 @@ MainWindow::MainWindow() :
 	m_workspace->setHorizontalScrollBarPolicy( Qt::ScrollBarAsNeeded );
 	m_workspace->setVerticalScrollBarPolicy( Qt::ScrollBarAsNeeded );
 
-	hbox->addWidget( sideBar );
-	hbox->addWidget( splitter );
-
+	hbox->addWidget(sideBar);
+	hbox->addWidget(splitter);
+	if(sideBarOnRight)
```

Comment:
To match our coding style:
```suggestion
	if (sideBarOnRight)
```

Reply:
Also, can you add a comment which explains what this block does?